package.json

```diff
@@ -19,23 +19,18 @@
   "license": "MIT",
   "files": ["dist", "src"],
   "source": "src/index.ts",
-  "module": "./dist/index.module.js",
   "main": "./dist/index.cjs",
   "types": "./dist/index.d.ts",
   "exports": {
     ".": {
-      "import": {
-        "types": "./dist/index.d.ts",
-        "default": "./dist/index.module.js"
-      },
-      "require": {
-        "types": "./dist/index.d.ts",
-        "default": "./dist/index.cjs"
-      }
-    }
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    },
+    "./package.json": "./package.json"
   },
   "scripts": {
-    "build": "microbundle --target node --sourcemap false --format cjs,esm"
+    "build": "microbundle --target node --sourcemap false --format cjs,modern"
```

Comment:
`esm` is ES5, switching to `modern` saves a tiny bit of build time (fewer transforms) and is completely safe behind `package.json#exports`. If you do want to use `package.json#module` for some reason, it could make sense to switch this back to `esm` in the unlikelihood this plugin is used in an env that existed between 2015-2017.